package.json

```diff
@@ -284,5 +287,10 @@
       "atom",
       "snapshotResult"
     ]
+  },
+  "devDependencies": {
+    "electron": "11.5.0",
+    "electron-builder": "^23.1.0",
+    "electron-rebuild": "^3.2.7"
   }
```

Comment:
These are going to break the snapshot script. They should be moved to `script/package.json`